scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -41,6 +83,111 @@ namespace <%= namespace %>.SDK.iOS {
             });
         }
 
+        private static bool IsShippingAddressField(string field) {
+            return ShippingAddressFieldsSet.Contains(field);
+        }
+
+        private static bool IsContactField(string field) {
+            return ShippingContactFieldsSet.Contains(field);
+        }
+
+        private static bool IsPartialShippingAddressField(string field) {
+            return PartialShippingAddressFieldsSet.Contains(field);
+        }
+
+        private static UpdateRequestStatus GetUpdateRequestStatusFromCheckoutUserErrors(List<UserError> errors) {
+
+            var payErrors = new List<ApplePayError>();
+
+            // Check to see if any of the user errors are not billing address errors
+            // If it is not a billing address error return a failure
+            foreach (var error in errors) {
+                var fields = error.field();
+                if (fields.Contains("billingAddress")) {
+                    string description = error.message();
+                    string lastField = fields[fields.Count - 1];
```

Comment:
We could put it into the try as well, but it'll fail on `fields.Contains("billingAddress")`